xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.h

```diff
@@ -26,8 +29,17 @@ class CDVDSubtitlesLibass
   */
   void Configure();
 
-  ASS_Image* RenderImage(int frameWidth, int frameHeight, int videoWidth, int videoHeight, int sourceWidth, int sourceHeight,
-                         double pts, int useMargin = 0, double position = 0.0, int* changes = NULL);
+  ASS_Image* RenderImage(int frameWidth,
```

Comment:
the method has 11 arguments it's usually a sign something is not good. Maybe combine some of the arguments into some "options" struct and pass it instead

Reply:
done